### PR TITLE
Project 2: Components.

### PR DESCRIPTION
**Pull request by Nelitaa**
**Project requirements**
In the previous project, I worked on the setup of a React application using Create React App (CRA.

For this project I follow these requirements:

- Delete all the boilerplate from CRA (text, images, styles).
- Create a directory called components.
- Inside components, create a new Calculator.js file.
- In Calculator.js, create a React component that matches this design:

calculator

For this project I followed the following recommendations:

- Don't add any extra design.
- Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- Use class based components (All components should be class-based).
- Once the Calculator is ready, import it and use it in your main component (App.js).
- Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div>
+      <MyCalculator />
     </div>
   );
```
Comment (Felix45) [inline src/App.js]: - [ ] Please consider changing this functional component to a `class` component, the project requirements for this project require that all components should be class-based components. Please see the screenshot below.

Screenshot from 2022-11-30 19-25-19

You can do this as shown below
```JavaScript
class App extends Component {
  render() {
    return {
      <Calculator />
    };
  }
}
```

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Nelitaa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights :heavy_check_mark: 
- [ ] All linter checks are set up and are passing :white_check_mark: 
- [ ] Good pull request title and summary :white_check_mark: 
- [ ] Good descriptive commit messages :white_check_mark: 

almost-there


## Required Changes ♻️
- [ ] Please consider removing some files generated by the `create-react-app` command from your project since they are not being used by your project. This makes your repository untidy, these files are `setUpTests.js`, `reportWebVitals.js`, and `App.test.js`

Screenshot from 2022-12-01 00-47-58


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (Nelitaa) [reply src/App.js]: Thank you for your review, it help me very much to understand better! 👍

**Review by FrederickMih — APPROVED** [review]
## Hi :raising_hand_man:  @Nelitaa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Calculator component matches the given design :+1: 
- Boilerplate from CRA has been removed :+1: 
- Descriptive PR :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**